Alcatraz/Controllers/ATZPackageTableViewDelegate.m

```diff
@@ -156,73 +146,23 @@ - (NSImage *)tableView:(NSTableView*)tableView websiteImageForTableColumn:(NSTab
     return [[Alcatraz sharedPlugin].bundle imageForResource:websiteImageName];
 }
 
-- (NSImage *)tableView:(NSTableView*)tableView packageTypeImageForTableColumn:(NSTableColumn *)tableColumn row:(NSInteger)row {
-    NSString* typeImageName = nil;
-    ATZPackage* package = [self tableView:tableView objectValueForTableColumn:tableColumn row:row];
-    if ([package isKindOfClass:[ATZPlugin class]]) {
-        typeImageName = @"740-gear";
-    } else if ([package isKindOfClass:[ATZTemplate class]]) {
-        typeImageName = @"808-documents";
-    } else if ([package isKindOfClass:[ATZColorScheme class]]) {
-        typeImageName = @"837-palette";
-    } else {
-        return nil;
-    }
-    if ([package isInstalled])
-        typeImageName = [NSString stringWithFormat:@"%@-selected", typeImageName];
-
-    return [self selectionTemplateImageNamed:typeImageName];
+- (void)loadImagesForVisibleRowsInTableView:(NSTableView *)tableView {
```

Comment:
Oh...actually I deleted these lines, as they were unused :open_mouth: 
I don't like to keep dead code in project.